### PR TITLE
strings cleanup #1: formatting and sorting

### PR DESCRIPTION
**Pull request by tralezab**
## About The Pull Request

* sorted most json files alphanumerically.
  * skipped memories, and wound files, and tcg as they became harder to read when alphanum sorted.
  * any other files i should undo sorting on, lmk.
* corrected any json files that were sorted via spaces instead of tabs

## Why It's Good For The Game

i want to start cleaning up the stings folder in general. it's really unsorted and formatting was all over the place. this handles the latter part of my issues with the folder

## Changelog

Not player facing

### CODE REVIEW

File: strings/tcg/set_one.json [hunk 1]
```diff
@@ -1,1506 +1,1506 @@
...
+	],
+	"templates": [
+		{
+			"icon": "icons/runtime/tcg/default.dmi",
 			"series": "coreset2020",
-			"rarity": "uncommon"
+			"template": "default"
 		}
 	]
 }
```
Comment (LemonInTheDark) [inline strings/tcg/set_one.json:1506]: This should not be at the bottom
In general the card files are setup in a way that semi defies sorting, might be best to just let them alone

Comment (tralezab) [reply strings/tcg/set_one.json:1506]: agreed, will reverse

**Comment by tralezab** [comment]
(does nothing for 10 days)

File: strings/clown_nonsense.json [hunk 2]
```diff
@@ -1,38 +1,35 @@
...
+		"HONK",
+		"HONK!",
+		"HONK!!!",
+		"HENK",
+		"HOOOOOOONK"
+	],
+	"non-honk-clown-words": [
+		"bananas",
+		"chapiteau",
+		"entrée",
```
Comment (LemonInTheDark) [inline strings/clown_nonsense.json:13]: This also seems like an ordering that was intentional, though I'm not sure if it's useful or not

File: strings/phobia.json [hunk 3]
```diff
@@ -1,31 +1,115 @@
...
+		"ayy lmao",
+		"greys",
+		"resin",
+		"mothership",
+		"abductee",
+		"xenobiology",
+		"jelly",
+		"slime",
+		"facehugger",
+		"lusty"
```
Comment (LemonInTheDark) [inline strings/phobia.json:17]: did you intend to sort the contents of these lists?

**Review by LemonInTheDark — CHANGES_REQUESTED** [review]
I don't think just by default alpha numeric sorting is best. It's useful in some places, but we often want to order files with some other form of intent.
I'm also a bit confused about how some of these were actually sorted, are you sorting lists?

**Comment by tralezab** [comment]
honestly i should pick this up when i have more time, sorry

**Pull request closed without merging**